docker-compose.yml

```diff
@@ -15,13 +15,13 @@ services:
 
   php:
     environment:
-      COMPOSER_AUTH: ${COMPOSER_AUTH}
```

Comment:
I am passing the `COMPOSER_AUTH` as an env flag here: https://github.com/dbergunder/docker-php-boilerplate/blob/master/.env.dist#L7 per the docs: https://getcomposer.org/doc/03-cli.md#composer-auth

Explanation:
https://www.previousnext.com.au/blog/managing-composer-github-access-personal-access-tokens#what-about-automated-testing-environments

Reply:
Ok I'll try the boilerplate from scratch to test this... I am getting the following unless it's in the args block, but maybe a difference in Docker config here we are dealing with.

<img width="693" alt="Screen Shot 2020-05-10 at 10 08 48 PM" src="https://user-images.githubusercontent.com/25436216/81520904-39b04c80-930b-11ea-8193-96c055c33b3b.png">

Reply:
Looks like your token is malformed, are you sure it's all on one line as a json string?  You know what, this might be from a poor example in the env.dist file.  Try without surrounding the json object with quotes:
```{"github-oauth":{"github.com":""}}```